README.md

```diff
@@ -2,7 +2,6 @@
 
 ![Unit tests and coverage](https://github.com/CSCfi/metadata-submitter-frontend/workflows/Unit%20tests%20and%20coverage/badge.svg)
 ![Code style check](https://github.com/CSCfi/metadata-submitter-frontend/workflows/Code%20style%20check/badge.svg)
-![Static type check](https://github.com/CSCfi/metadata-submitter-frontend/workflows/Static%20type%20check/badge.svg)
```

Comment:
maybe we can reintroduce the badge, no need to delete it now

Reply:
Sounds good to me.